x-pack/plugins/cases/public/components/create/form_context.test.tsx

```diff
@@ -777,14 +554,16 @@ describe.skip('Create case', () => {
     userEvent.click(screen.getByTestId('create-case-submit'));
 
     await waitFor(() => {
-      expect(afterCaseCreated).toHaveBeenCalledWith(
-        {
-          id: sampleId,
-          ...sampleDataWithoutTags,
-        },
-        createAttachments
-      );
+      expect(afterCaseCreated).toHaveBeenCalled();
     });
+
```

Comment:
What was the reasoning for this split?

Reply:
As we discussed offline, the reasoning is that the connectors' tests in `form_context.test.tsx` are slow and cause timeouts. To mitigate the problem I left one connector, to test the whole integration with the create case form, and moved the specific tests to each connector.

Reply:
I though that it maybe faster, maybe mistakenly, to wait for the callback to be called and then check the arguments.

Reply:
Ah, I was wondering if by wrapping `toHaveBeenCalledWith` in a `waitFor`, if the call never comes we timeout rather than fail.

Reply:
If it was called it should be called with the correct arguments. The timeout error does not tell you why it failed. If we fail we can see the difference and debug better.